### ISSUE TITLE
While I am working on note A, suddenly and unexpectedly the UI jumps to random unrelated notes B, C, D etc

### ISSUE THREAD

**Issue by dejafait** [laurent22/joplin#11566]
### Operating system

Android

### Joplin version

3.2.3

### Current behaviour

I assume the bug happens during background sync and typically while working on two devices or more simultaneously.

Sometimes while I am working on note A, my interaction triggers a background sync (it's just my guess though) and the UI suddenly and very quickly switches through a succession of other notes (I guess while processing the sync changes made from my other device). I end up on a totally different note than A.

This is very disturbing and confusing when this happens.

I spent several hours trying to find a reproducible flow and unfortunately could not find one. There is something random about it that I can't pinpoint.

It's still worth reporting imho because even if the cause is unclear the expected fix is obvious (when note A is open, stay on it).

### Expected behaviour

When note A is open (either viewing of editing), stay on it. Don't randomly jump to unrelated notes.

**Comment by mrjo118** [comment]
Possibly related to this?
https://github.com/laurent22/joplin/issues/11479

Have you moved any notes to different notebooks as part of the flow which caused the issue?

**Comment by dejafait** [comment]
Yes, I frequently move notes in and out of current notebook, also created and updated notes. I confirm this seems like it is the same issue.

**Comment by personalizedrefrigerator** [comment]
I'm closing this as a duplicate of https://github.com/laurent22/joplin/issues/11479.

Thank you for reporting this!
